ui/src/hooks/contracts/sds.ts

```diff
@@ -0,0 +1,649 @@
+      { indexed: false, internalType: 'uint32', name: 'microLinkPerEth', type: 'uint32' },
+      { indexed: false, internalType: 'uint32', name: 'linkGweiPerObservation', type: 'uint32' },
+      { indexed: false, internalType: 'uint32', name: 'linkGweiPerTransmission', type: 'uint32' },
+    ],
+    name: 'BillingSet',
+    type: 'event',
+  },
+  { anonymous: false, inputs: [], name: 'CheckAccessDisabled', type: 'event' },
+  { anonymous: false, inputs: [], name: 'CheckAccessEnabled', type: 'event' },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: false, internalType: 'uint32', name: 'previousConfigBlockNumber', type: 'uint32' },
+      { indexed: false, internalType: 'uint64', name: 'configCount', type: 'uint64' },
+      { indexed: false, internalType: 'address[]', name: 'signers', type: 'address[]' },
+      { indexed: false, internalType: 'address[]', name: 'transmitters', type: 'address[]' },
+      { indexed: false, internalType: 'uint8', name: 'threshold', type: 'uint8' },
+      { indexed: false, internalType: 'uint64', name: 'encodedConfigVersion', type: 'uint64' },
+      { indexed: false, internalType: 'bytes', name: 'encoded', type: 'bytes' },
+    ],
+    name: 'ConfigSet',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      {
+        indexed: true,
+        internalType: 'contract LinkTokenInterface',
+        name: '_oldLinkToken',
+        type: 'address',
+      },
+      {
+        indexed: true,
+        internalType: 'contract LinkTokenInterface',
+        name: '_newLinkToken',
+        type: 'address',
+      },
+    ],
+    name: 'LinkTokenSet',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'uint256', name: 'roundId', type: 'uint256' },
+      { indexed: true, internalType: 'address', name: 'startedBy', type: 'address' },
+      { indexed: false, internalType: 'uint256', name: 'startedAt', type: 'uint256' },
+    ],
+    name: 'NewRound',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'uint32', name: 'aggregatorRoundId', type: 'uint32' },
+      { indexed: false, internalType: 'int192', name: 'answer', type: 'int192' },
+      { indexed: false, internalType: 'address', name: 'transmitter', type: 'address' },
+      { indexed: false, internalType: 'int192[]', name: 'observations', type: 'int192[]' },
+      { indexed: false, internalType: 'bytes', name: 'observers', type: 'bytes' },
+      { indexed: false, internalType: 'bytes32', name: 'rawReportContext', type: 'bytes32' },
+    ],
+    name: 'NewTransmission',
+    type: 'event',
+  },
+  {
+    anonymous: false,
+    inputs: [
+      { indexed: true, internalType: 'address', name: 'transmitter', type: 'address' },
+      { indexed: true, internalType: 'address', name: 'payee', type: 'address' },
+      { indexed: false, internalType: 'uint256', name: 'amount', type: 'uint256' },
+      {
```

Comment:
Do we need this abi file if we import directly from the contracts?

Reply:
Ah pls ignore this looks like chainlink

Reply:
```
{
    inputs: [],
    name: 'latestAnswer',
    outputs: [{ internalType: 'int256', name: '', type: 'int256' }],
    stateMutability: 'view',
    type: 'function',
  },
```
```
  const snxContract = new Contract(snxPoolAddressMainnet, snxAbi, globalProviders.mainnet);
      const sdsContract = new Contract(sdsPoolAddressMainnet, sdsAbi, globalProviders.mainnet);

      const [snxPriceBn, sdsPriceBn] = await Promise.all([
        snxContract.latestAnswer(),
        sdsContract.latestAnswer(),
      ]);
```
  Need to call `latestAnswer`